apps/crn-server/src/data-providers/contentful/analytics.data-provider.ts

```diff
@@ -32,76 +34,94 @@ export class AnalyticsContentfulDataProvider implements AnalyticsDataProvider {
             displayName: team.displayName || '',
             interestGroupLeadershipRoleCount: getUniqueIdCount(
```

Comment:
I think it's missing the `!team.inactiveSince` here like there is for `interestGroupMemberCount`